app/assets/javascripts/select_change_label.js

```diff
@@ -0,0 +1,24 @@
+jQuery(function () {
+  $( document ).ready(function() {
+    $('[label_target]').each(function() {
+      change_label($(this));
+    })
+  });
+
+  $(document).on('change', '[label_target]', function () {
+    change_label($(this));
+  })
+
+  function change_label(element) {
+    var $id_label = element.attr('class_form').toLowerCase().concat('_', element.attr('label_target'));
+    var $label_target = $(`label[for="${$id_label}"]`);
+    if (element.find(":selected").attr('value') == element.attr('value_change_label')){  
```

Comment:
```suggestion
    if (element.find(':selected').val() === element.attr('value_change_label')) {  
```
No se si funciona `val()` ahí, si no es así volvelo a como estaba... jeje.